app/helpers/emails_helper.rb

```diff
@@ -31,8 +31,8 @@ def action_title(url)
   end
 
   def color_email_diff(diffcontent)
-    formatter = Rugments::Formatters::HTML.new(cssclass: "highlight", inline_theme: :github)
-    lexer = Rugments::Lexers::Diff.new
+    formatter = Rouge::Formatters::HTML.new(cssclass: "highlight", inline_theme: :github)
```

Comment:
I don't think so. It just led to confusion because I was trying to debug a failed test, and my debugger pointed me towards `html_inline.rb` instead of `html.rb`. Apparently, the use of the symbol `:github` as opposed to `github` caused this error.